sdk/lib/delayimp/CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 
 add_definitions(-DUNICODE -D_UNICODE)
-add_library(delayimp delayimp.c)
+add_library(delayimp
+    delayimp.c
+    pfnDliFailureHook2.c
+    pfnDliNotifyHook2.c
```

Comment:
Why requiring two separate files for each single external variable?

Reply:
So you can define 1 in your module and import the other one from the lib.